packages/osc-ui/src/components/Button/button.scss

```
@@ -187,6 +187,21 @@
                 )
             )
         ),
+        septenary: (
+            bg-color: transparent,
+            bg-hover-color: var(--btn-hover-color-tertiary),
+            fg-color: var(--color-tertiary),
+            fg-hover-color: var(--btn-fg-hover-color-tertiary),
+            font-weight: $fw-reg,
+            text-transform: capitalize,
+            border-bottom: 1px solid var(--color-tertiary),
+            sizes: (
+                lg: (
+                    padding: $space-l $space-3xl,
+                    font-size: $fs-m
```

I think the font is too big as well, looking at the design it should be `16px`, so you can use `--font-scale-base` in this case to bring it to `1rem`
```suggestion
       font-size: var(--font-scale-base);
```